### PR TITLE
feat: hide toolbar on mobile when not used

### PR DESCRIPTION
**Pull request by y-lohse**

### CODE REVIEW

**Comment by y-lohse** [comment]
[(2) [forum-4529] [PHO] Afficher les actions de la visionneuse au "touch" sur mobile](https://trello.com/c/8wNVJQQL/361-2-forum-4529-pho-afficher-les-actions-de-la-visionneuse-au-touch-sur-mobile)

File: src/components/Viewer.jsx [hunk 1]
```diff
@@ -66,11 +76,26 @@ export class Viewer extends Component {
     this.setState({ isImageLoading: false })
   }
 
+  toggleToolbar () {
+    // invert the current state
+    const hidden = !this.state.hideToolBar
+    this.setState({ hideToolBar: hidden })
+    // if the toolbar is now visible, reset the hide delay
+    if (!hidden) this.hideToolbarAfterDelay()
```
Comment (enguerran) [inline src/components/Viewer.jsx]: Should we write this this way?

```js
this.setState(state => ({...state, hideToolBar: !state.hidden}))
```

and

```js
componentDidUpdate(prevState, prevProps) {
  if (!prevState.hideToolBar) this.hideToolbarAfterDelay()
}
```

File: src/components/Viewer.jsx [hunk 2]
```diff
@@ -66,11 +76,26 @@ export class Viewer extends Component {
...
+  }
+
+  hideToolbarAfterDelay () {
+    clearTimeout(this.hideToolbarTimeout)
+    this.hideToolbarTimeout = setTimeout(() => {
+      this.setState({ hideToolBar: true })
+    }, TOOLBAR_HIDE_DELAY)
+  }
```
Comment (enguerran) [inline src/components/Viewer.jsx]: sometimes you write `ToolBar` sometimes `Toolbar`. Could we be more consistent? 😉

Comment (y-lohse) [inline src/components/Viewer.jsx]: Oh, good point.

File: src/components/Viewer.jsx (hunk 1 above)
Comment (y-lohse) [inline src/components/Viewer.jsx]: Thanks for the link! It's a very interesting difference I wasn't aware of.

**Pull request merged**